docs/man.rst

```diff
@@ -26,178 +26,154 @@ To install the Quipucords server and command line interfaced (CLI) client:
 
 The following sections describe these commands, their subcommands, and their options in more detail. They also describe additional tasks that are not highlighted in the previous list of major workflow tasks.
 
-Installation
-------------
-The ``qpc-tools install`` command with no options performs a basic installation with the preset defaults. However, it is recommended the ``qpc-tools install`` command with options to change default username and passwords. The most common scenarios where you might use options to change the basic installation process are explained in the `Installing offline`_, `Installing a specific version`_, and `Installing the server and command line interface separately`_ sections.
+Server
+------
 
-Note that in log information for the ``qpc-tools install`` command, references to ``quipucords server`` are relevant to the Quipucords server, and references to ``qpc CLI`` are relevant to the Quipucords command line interface client.
+Server Installation
+^^^^^^^^^^^^^^^^^^^
+The ``qpc-tools server install`` command with no options performs a basic installation with the preset defaults. However, it is recommended the ``qpc-tools server install`` command with options to change default username and passwords. The most common scenarios where you might use options to change the basic installation process are explained in the `Installing the server offline`_ and  `Installing a specific version of the server`_ sections.
```

Comment:
change `However, it is recommended the "qpc-tools...` to `However it is recommended to run the "qpc-tools` 

- also wouldn't the default fail? I thought that we made passwords required.

Reply:
Good catch